tests/components/hue/test_config_flow.py

```diff
@@ -162,14 +243,13 @@ async def test_flow_two_bridges_discovered_one_new(hass, aioclient_mock):
     result = await hass.config_entries.flow.async_init(
         const.DOMAIN, context={"source": "user"}
     )
+
     assert result["type"] == "form"
-    assert result["step_id"] == "link"
-    flow = next(
-        flow
-        for flow in hass.config_entries.flow.async_progress()
-        if flow["flow_id"] == result["flow_id"]
-    )
-    assert flow["context"]["unique_id"] == "beer"
+    assert result["step_id"] == "init"
+    assert result["data_schema"]({"id": "beer"})
+    assert result["data_schema"]({"id": "manual"})
+    with pytest.raises(vol.error.MultipleInvalid):
+        assert not result["data_schema"]({"id": "bla"})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5795327)